### PR TITLE
Removed Unnecessary Requirements

### PR DESCRIPTION
**Pull request by clintonb**
XCOM-340

@jimabramson @rlucioni

### CODE REVIEW

**Comment by jimabramson** [comment]
:+1:

File: requirements/base.txt [hunk 1]
```diff
@@ -1,10 +1,7 @@
 analytics-python==1.0.3
 Django==1.7.7
 django-appconf==0.6
-django_compressor==1.4
```
Comment (rlucioni) [inline requirements/base.txt:4]: Isn't this required by the refund list and detail views?

Comment (rlucioni) [inline requirements/base.txt:4]: https://github.com/edx/ecommerce/blob/master/ecommerce/settings/base.py#L244

Comment (rlucioni) [inline requirements/base.txt:4]: Resolved offline. django_compressor is included with django-oscar.

**Pull request merged**